wremnants/syst_tools.py

```diff
@@ -0,0 +1,21 @@
+import hist
+import numpy as np
+
+def scale_hist_to_variations(scale_hist):
```

Comment:
Well a similar thing needs to be done for the scale hist that don't have the coefficients included, so maybe you should reserve this name for that and add something about helicity here